packages/amplication-git-pull-request-service/src/core/diff/visitors/delete-files.ts

```diff
@@ -0,0 +1,24 @@
+import { Difference } from 'dir-compare';
+import { join } from 'path';
+import { PrModule } from '../../../constants';
+
+export function deleteFilesVisitor({
+  state,
+  name1,
+  type1,
+  type2,
+  relativePath,
+}: Difference): PrModule | null {
```

Comment:
```suggestion
export function deleteFilesVisitor(diff: Difference): PrModule | null {
const { state, name1, type1, type2, relativePath } = diff;
```
I think it would be a lot cleaner to perform destructuring inside the function block.